client/me/purchases/upcoming-renewals/upcoming-renewals-dialog.tsx

```diff
@@ -84,6 +84,7 @@ const UpcomingRenewalsDialog: FunctionComponent< Props > = ( {
 	onConfirm,
 	submitButtonText = '',
 	showManagePurchaseLinks = true,
+	getManagePurchaseUrl = managePurchase,
```

Comment:
Since this is typescript, these props are managed by the `Props` interface above and you'll need to add `getManagePurchaseUrl` there too. It's kind of like PropTypes, but you'll have to set it to something like `getManagePurchaseUrl: ( siteSlug: string, purchaseId: number ) => string`... although it might work if you do `getManagePurchaseUrl: typeof managePurchase`?

Reply:
aah ok. Got it. I was about to add it there but wasn't sure. I'll add it in.